### PR TITLE
Add blog post for Q1 2023 update

### PR DESCRIPTION
**Pull request by choldgraf**
This adds a short q1 2023 update for our blog!

- part of: https://github.com/2i2c-org/team-compass/issues/685

### CODE REVIEW

File: content/blog/2023/2023-q1-update/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+- We onboarded several new community colleges to our "JupyterHubs Education in Community Colleges" collaboration with [CloudBank](https://www.cloudbank.org/welcome-cloudbank) and [UC Berkeley CDSS](https://data.berkeley.edu/).
+
+## Service improvements
+
+Below are a few highlights for ways in which we improved our [Managed Cloud Service](https://docs.2i2c.org) for our partner communities.
+
+### We simplified our authentication workflow with CILogon
+
+Authentication services allow us to identify a user when they log onto a hub, which determines their ability to access hub resources.
+Previously we had used a combination of [Auth0](https://auth0.com/) and [CILogon](https://www.cilogon.org/) for authentication.
```
Comment (damianavila) [inline content/blog/2023/2023-q1-update/index.md]: And GitHub as well (https://infrastructure.2i2c.org/hub-deployment-guide/configure-auth/github-orgs.html), but not sure if we need to mention it in this context.

File: content/blog/2023/2023-q1-update/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+We tend to work with _leaders_ of communities that utilize our service and infrastructure for many others in their network.
+For example, a teacher with a classroom of students, or a researcher with a global network of collaborators.
+
+In these cases, it's useful to track how many users are actively using a platform over various metrics of time.
+This can tell you whether your community finds a service useful, and whether this is growing or shrinking.
+
+We use [Grafana](https://grafana.com/) to automatically generate dashboards of activity for all of our community hubs and clusters.
+However, tracking **daily, weekly, and monthly active users** was not part of JupyterHub's core functionality.
+
+So, we decided to upstream this functionality into JupyterHub and expose it via [the JupyterHub Grafana project](https://github.com/jupyterhub/grafana-dashboards).
```
Comment (damianavila) [inline content/blog/2023/2023-q1-update/index.md:60]: "we decided to upstream this functionality", maybe we may want to add a link to Yuvi's blog post in the Jupyter blog: https://blog.jupyter.org/accurately-counting-daily-weekly-monthly-active-users-on-jupyterhub-6fbec6c6ce2f

Comment (damianavila) [reply content/blog/2023/2023-q1-update/index.md:60]: jaja... forget about it, I just saw the link below...

File: content/blog/2023/2023-q1-update/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+{{< figure src="support-button.png" width=500 title="Our fancy new support button!" >}}
+
+
+### We upgraded Kubernetes across all of our AWS clusters
+
+[Kubernetes](https://kubernetes.org) is at the core of our cloud infrastructure and scalability.
+We use either a shared or a dedicated Kubernetes cluster for each of our community partners, and it is the foundation upon which all of their Jupyter infrastructure rests.
+
+One of the biggest challenges with managing an ongoing cloud service is keeping the underlying infrastructure upgraded.
+This brings in new stability and functionality, but also often involves manual steps and toil
```
Comment (damianavila) [inline content/blog/2023/2023-q1-update/index.md]:
```suggestion
This brings in new stability and functionality, but also often involves manual steps and toil.
```

**Review by damianavila — APPROVED** [review]
Minor comment, otherwise, LGTM!
It is quite nice to read all the things we have done in Q1 (and this is just a summary 😉)!

File: content/blog/2023/2023-q1-update/index.md (hunk 1 above)
Comment (choldgraf) [inline content/blog/2023/2023-q1-update/index.md]:
```suggestion
Previously we had used a combination of [Auth0](https://auth0.com/), [CILogon](https://www.cilogon.org/), or [GitHub](https://infrastructure.2i2c.org/hub-deployment-guide/configure-auth/github-orgs.html) for authentication.
```

**Comment by choldgraf** [comment]
Forgot to actually hit send on this one, will send out now

**Pull request merged**